apps/web/playwright/webhook.e2e.ts

```diff
@@ -577,10 +581,28 @@ test.describe("MEETING_ENDED, MEETING_STARTED", async () => {
     expect(newMeetingEndedTriggers.length).toBe(0);
 
     // disable webhook
-    const submitPromise = page.waitForResponse("/api/trpc/webhook/edit?batch=1");
-    await page.getByTestId("webhook-switch").click();
-    const response = await submitPromise;
-    expect(response.status()).toBe(200);
+    await submitAndWaitForResponse(page, "/api/trpc/webhook/edit?batch=1", {
+      action: () => page.getByTestId("webhook-switch").click(),
+    });
+
+    const scheduledTriggersAfterDisabling = await prisma.webhookScheduledTriggers.findMany({
+      where: {
+        webhook: {
+          userId: user.id,
+        },
+      },
+      select: {
+        payload: true,
+        webhook: {
+          select: {
+            userId: true,
+          },
+        },
+        startAfter: true,
+      },
+    });
+
+    expect(scheduledTriggersAfterDisabling.length).toBe(0);
```

Comment:
Do we need this? We are already checking the response, and then we are making an extra db query just to check the length.

Reply:
will remove

Reply:
We're checking that the toggle actually removes it from the DB. Also the DB query is running in the same server so no really a perf cost there.